test/authorize-specs.js

```diff
@@ -0,0 +1,52 @@
+import { authorize } from '../lib/authorize.js';
+import 'mochawait';
+import * as teen_process from 'teen_process';
+import * as xcode from 'appium-xcode';
+import path from 'path';
+import sinon from 'sinon';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+
+let teenProcessMock = null;
```

Comment:
Can have a look here: https://github.com/appium/appium-doctor/blob/sebv-doctor-who/test/mock-utils.js to make it more terse, ask @moizjv